compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/Passes.td

```diff
@@ -94,4 +94,13 @@ def ConvertAttentionToOnlineAttentionPass :
   let summary = "Converts attention op to online_attention op";
 }
 
+def FoldUnitExtentDimsPass :
+      Pass<"iree-linalg-ext-fold-unit-extent-dims", "">  {
+  let summary = "Folds unit dims for iree_linalg_ext ops";
+  let dependentDialects = [
+    "::mlir::tensor::TensorDialect",
+    "::mlir::iree_compiler::IREE::LinalgExt::IREELinalgExtDialect"
+  ];
+}
```

Comment:
I think you should document that it is done by slicing somewhere. Because there is another approach which is using reshapes to fold unit dims.